src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netstandard2.0;net45;net461</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard2.0</TargetFrameworks>
-    <Version>0.7.1-beta</Version>
```

Comment:
intended to go away?

Reply:
Consolidated three of these into [one file](https://github.com/DataDog/dd-trace-dotnet/pull/278/files#diff-6483fde6c45c90fdfe98ca72eadfa651R5). Originally we thought we might have different versions for the 3 libraries, so we kept the version numbers separate (for example, if only one library suffers changes). So far we've kept them in sync, and will keep doing so for now, so this way it's a little cleaner.